docs/en/latest/plugins/http-dubbo.md

```diff
@@ -0,0 +1,127 @@
+| read_timeout             | number  | False    | 6000    |              | Upstream tcp read_timeout                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| send_timeout             | number  | False    | 6000    |              | Upstream tcp send_timeout                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+
+## Enable Plugin
+
+The example below enables the `http-dubbo` Plugin on the specified Route:
+
+:::note
+You can fetch the `admin_key` from `config.yaml` and save to an environment variable with the following command:
+
+```bash
+admin_key=$(yq '.deployment.admin.admin_key[0].key' conf/config.yaml | sed 's/"//g')
+```
+
+:::
+
+```shell
+curl -i http://127.0.0.1:9180/apisix/admin/routes/1  \
+-H "X-API-KEY: $admin_key" -X PUT -d '
+{
+    "uri": "/TestService/testMethod",
+    "plugins": {
+      "http-dubbo": {
+      "method": "testMethod",
+      "params_type_desc": "Ljava/lang/Long;Ljava/lang/Integer;",
+      "serialized": true,
+      "service_name": "com.xxx.xxx.TestService",
+      "service_version": "0.0.0"
+    }
+    },
+    "upstream": {
+        "type": "roundrobin",
+        "nodes": {
+            "127.0.0.1:20880": 1
+        }
+    }
+}'
+```
+
+## Example usage
+
+Once you have configured the Plugin as shown above, you can make a request as shown below:
+
+```shell
+curl --location 'http://127.0.0.1:9080/TestService/testMethod' \
+--data '1
+2'
+```
+
```

Comment:
Is there an expected response that you could add here?

Reply:
Sorry, it's not possible to add a return value example here. In the documentation demo, we only defined the method's input parameters without defining the return value signature. Providing a return value example would not be rigorous. I believe it's not necessary.